src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -22,6 +28,9 @@
     private final int moveSpeed = 200;
     private int posY = 0;
     private final Color blueColor = new Color(0, 0, 1, 1);
+    private ColonyView colonyView;
+    private Colony colony;
```

Comment:
`GameScreen` should not know about `ColonyView` or `Colony`. 

`ColonyView` should implement `IView` and be added to the list of `IView`s passed to `GameScreen` through the constructor. 

`Colony` should not be in here at all.

Reply:
I'll remove it. It was only for the demo.